utility/BUILD.gn

```diff
@@ -1,61 +0,0 @@
-import("//brave/components/brave_ads/browser/buildflags/buildflags.gni")
```

Comment:
let's keep this with `group("utility") { public_deps = [ "//chrome/utility" ] }` as discussed in dm

Reply:
Good point. I've changed the PR to do this now.

@bridiver PTAL when you have a moment, thanks!